jlh/JonFirstApp/app/src/main/java/com/e_conomic/jonfirstapp/DisplayMessageFragment.java

```diff
@@ -1,17 +1,49 @@
 package com.e_conomic.jonfirstapp;
 
 import android.os.Bundle;
+import android.support.annotation.Nullable;
 import android.support.v4.app.Fragment;
 import android.view.LayoutInflater;
 import android.view.ViewGroup;
 import android.view.View;
+import android.widget.TextView;
+
 
 public class DisplayMessageFragment extends Fragment {
 
+    // The message to be displayed.
+    private String message = "";
+
+    // The TextView that displays the message.
+    private TextView display_message;
+
+    @Override
```

Comment:
It lets the compiler (and IDE I guess?) know that the following method should override a method in the superclass? :)